migrator.py

```diff
@@ -17,16 +17,16 @@ def migrator_parser():
     parser = argparse.ArgumentParser(
         description='Define bio_covid database and insert data by calling separate migrate scripts.')
     parser.add_argument("-n", "--num_threads", type=int,
-                        help="Number of threads to enable multi-threading (default: 8)", default=8)
-    parser.add_argument("-c", "--commit_batch", help="Sets the number of queries made per commit (default: 50)",
-                        default=50)
-    parser.add_argument("-d", "--database", help="Database name (default: bio-covid)", default="bio_covid")
+                        help="Number of threads to enable multi-threading (default: %(default)s)", default=8)
+    parser.add_argument("-c", "--commit_batch", type=int,
+                        help="Sets the number of queries made per commit (default: %(default)s)", default=50)
```

Comment:
If this parameter was ever passed in, it was parsed as a string, and since an integer is never equal to string, no batching took place.